### PR TITLE
AC-165 Added card backgrounds to listviews and added onpress color

### PR DESCRIPTION
**Pull request by AvijitGhosh82**

### CODE REVIEW

**Comment by AvijitGhosh82** [comment]
Merged.

**Comment by brainbreaker** [comment]
@AvijitGhosh82 See these screenshots [1](http://s19.postimg.org/c0v6b29dv/12442770_988426351234125_2108521811_n.jpg) and [2](http://s19.postimg.org/egwzowrgj/12834518_988426371234123_2070022489_n.jpg). I would suggest reverting to previous white background colour only, we should keep it simple. In my opinion, this colour is not at all intuitive. Also, cards are not visible plus I am not able to see any onpress colour or maybe I'm seeing just the onpress colour, please check that. I am using Android 4.4.4 KitKat.

**Comment by AvijitGhosh82** [comment]
Interesting... It works properly on Marshmallow. I do not have a KitKat device to test, I shall look into this immediately.

File: openmrs-client/src/main/res/color/cardhighlight.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/light_teal"
+        android:state_pressed="true" />
+    <item android:color="@color/card_background"
+        android:state_pressed="false" />
+</selector>
```
Comment (AvijitGhosh82) [inline openmrs-client/src/main/res/color/cardhighlight.xml]: @brainbreaker  Please try the following code and report back:

`<?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:color="@color/light_teal"
         android:state_pressed="true" />
    <item android:color="@color/light_teal"
         android:state_focused="true" />
     <item android:color="@color/card_background"/>
 </selector>`

**Comment by AvijitGhosh82** [comment]
@brainbreaker I have tagged you in an inline code commit, please apply the patch on your local device and report back if it is working.

File: openmrs-client/src/main/res/color/cardhighlight.xml (hunk 1 above)
Comment (brainbreaker) [inline openmrs-client/src/main/res/color/cardhighlight.xml]: @AvijitGhosh82 It has reverted the background colour to white but I am not seeing any onPress colour.

Comment (AvijitGhosh82) [inline openmrs-client/src/main/res/color/cardhighlight.xml]: Hmm. The code seems to be all right though, I have followed the exact code for the statelist color. Do you know what's wrong? I'll patch this change till then, but let us investigate.

Comment (brainbreaker) [inline openmrs-client/src/main/res/color/cardhighlight.xml]: Alright, working on it.

**Pull request merged**